core/dbt/include/global_project/macros/db_api/materialized_view/create.sql

```diff
@@ -0,0 +1,10 @@
+{% macro db_api__create_materialized_view_as(relation, sql) %}
+    {{ adapter.dispatch('db_api__create_materialized_view_as', 'dbt')(relation, sql) }}
+{% endmacro %}
+
+{% macro default_db_api__create_materialized_view_as(relation, sql) -%}
```

Comment:
A few things:

1. Typo: you need a __ after default.
2. We should go with a naming convention of __-separated path. So that would make this macro `db_api__materialized_view__create()` and `default__db_api__materialized_view__create()`.